build-and-test.ps1

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env pwsh
+[cmdletbinding()]
+param(
+    [ValidateSet('aspnet','wcf','samples')]
+    [string]$ManifestType,
+    [string]$VersionFilter = "*",
+    [string]$OSFilter = "*",
+    [string]$OptionalImageBuilderArgs,
+    [switch]$SkipTesting = $false
+)
+
+$manifestTypeExt = $ManifestType
+if (-not [string]::IsNullOrEmpty($manifestType)) {
```

Comment:
This can be simplified to `if ($manifestType) {`

Reply:
This is no longer applicable after the `manifest.runtime-sdk.json` refactoring.

Reply:
To be more specific, the code was removed.